package.json

```diff
@@ -67,10 +67,10 @@
     "@babel/core": "^7.19.3",
     "@babel/eslint-parser": "^7.19.1",
     "@folio/eslint-config-stripes": "^6.1.0",
-    "@folio/stripes": "^7.0.0",
+    "@folio/stripes": "^8.0.0",
```

Comment:
We should use
```
"@folio/stripes": "^7.0.0 || ^8.0.0",
```
so that we continue to support the old version of Stripes as well as the newer version.